docs/faq.rst

```diff
@@ -256,6 +256,18 @@ addresses of the users you wish to share with to give them permissions
 to your workflow. This is only available for hosted workflows and users
 with Google accounts linked to Terra.
 
+How do I add a specific version of a workflow to a collectionn and why would I want to?
+------------------------------------------------
+
+When creating a collection in an organization, you can pin either a specific version of a workflow or pin the workflow in general. This is a flexible system and can mean different things to different organizations. You can elaborate on what this means to you as an organization maintainer in the markdown description associated with a collection. 
```

Comment:
I don't feel too strongly about this, but elsewhere in the doc we "add" a workflow rather than "pin" it.